ubuntu.sh

```diff
@@ -236,8 +235,9 @@ ubuntu_grub_fix() {
     local volgroup=$(echo $line | cut -d " " -f9)
     local dmdevice=$(echo $line | cut -d "/" -f2)
 
-    rm $mapper/$volgroup
-    cp -R $FOLD/hdd/dev/$dmdevice $mapper/$volgroup
+    rm "$mapper/$volgroup"
```

Comment:
than make $mapper at least read-only (or refactor the whole ubuntu_grub_fix() because it's horrible (and not needed anymore?))